packages/eslint-plugin-expo/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
```

Comment:
I don't think I can because [monorepo](https://eslint.org/docs/latest/use/configure/configuration-files-new) - it seems like I'd have to update to using `eslint.config.js` at the root project, so that should probably be a separate PR